iron-form.html

```diff
@@ -139,7 +150,18 @@
       this._requestBot.withCredentials = this.withCredentials;
 
       if (this.method.toUpperCase() == 'POST') {
-        this._requestBot.body = JSON.stringify(json);
+        if (this.requestBody) {
+          this._requestBot.body = this.requestBody;
+        }
+        else if (this.contentType == 'application/json') {
+          this._requestBot.body = JSON.stringify(json);
+        } else {
+          // <iron-ajax> can only serialize the query string from the params
+          // property, so temporarily set this.
+          this._requestBot.params = json;
+          this._requestBot.body = this._requestBot.queryString;
+          this._requestBot.params = {};
```

Comment:
With https://github.com/PolymerElements/iron-ajax/pull/83 you should be able to just do:

```javascript
this._requestBot.body = json;
this._requestBot.contentType = this.contentType;
```

And it will automagically do all of this (i.e. it'll encode the body as either json or form-encoded based on the content type).